src/routes/safe/components/Settings/ManageOwners/AddOwnerModal/index.tsx

```diff
@@ -59,7 +61,8 @@ export const AddOwnerModal = ({ isOpen, onClose }: Props): React.ReactElement =>
   const [activeScreen, setActiveScreen] = useState('selectOwner')
   const [values, setValues] = useState<OwnerValues>({ ownerName: '', ownerAddress: '', threshold: '' })
   const dispatch = useDispatch()
-  const { address: safeAddress = '', currentVersion: safeVersion = '' } = useSelector(currentSafe) ?? {}
+  const { address: safeAddress = '' } = useSelector(currentSafe) ?? {}
```

Comment:
I wouldn't destructure the store, use the correct selector:
```
const safeAddress = useSelector(safeAddressFromUrl)
```

Reply:
Thanks! Lazy me didn't check if there was a selector only for the safeAddress 👍👍